section 4_Flask_Restfull/app.py

```diff
@@ -0,0 +1,59 @@
+from flask import Flask,request
+from flask_restful import Resource, Api,reqparse
+from flask_jwt import JWT,jwt_required
+from security import authenticate, identity
+
+app = Flask(__name__)
+app.secret_key = 'jose'
```

Comment:
Why jose?

Reply:
because of the author of the tutorial... But I need to change it something complicated in the next commit.